diagnostic.md

```diff
@@ -20,6 +26,11 @@ c = c + d;
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
 
+After the code executes c is equal to 5
+= is an assignment operator; it will assign whatever is on the right hand side
+  to what is on the left hand side (such as a variable)
++ is an addition operator; it will add whatever is on the right side to what
+  is on the left side, sometimes doing type conversions to do so.
```

Comment:
The value of `c` is actually equal to 7. The reason is that on line 23, `c` is assigned the value of `c + d`, which is 5 + 2.